### PR TITLE
Bypass e-mail subscriptions on download.

### PR DESCRIPTION
**Pull request by frank-trampe**

### CODE REVIEW

**Comment by ctrlcctrlv** [comment]
Yes! I noticed that the other day...we can just use Twitter and the mailing lists, I certainly am not going to volunteer to write a newsletter on top of all that. So glad this was removed. Not only that, collecting people's data sucks

**Comment by ctrlcctrlv** [comment]
Oh, I see, there was never a newsletter, it's just fontforge-announce. Derp.

**Comment by frank-trampe** [comment]
Collecting lots of data is bad, but we do need to get e-mail addresses somehow. This is likely to require switching e-mail providers, but Sourceforge does not allow exporting the lists, so we would probably need to stack the switch with a release announcement so that people read the message and subscribe to the new list.

**Comment by ctrlcctrlv** [comment]
> Sourceforge does not allow exporting the lists

@sourceforge, @brondsem That is seriously not cool. We spend years building up a list, we spend years getting people subscribed to it, then you just one day change how it works and hold our data hostage? That is no way to treat FOSS projects.

If we don't get an answer from them as to how we can get our own list members I'll raise as big a :hankey: as I can on Twitter @frank-trampe :-) I can't promise it will be a big :hankey: , but a :hankey: it shall be.

**Comment by frank-trampe** [comment]
I'm similarly upset about it. The newly added cross-origin-framing restriction is totally unnecessary since the subscription goes through only if the user clicks the link in the confirmation e-mail.

File: en-US/downloads/gnulinux.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ All we ask is that you subscribe to the FontForge announcement list so that we c
 
 <div id="emls_box">
 <form id="emls_vform" style="display: inline;" name="emls_vform" target="emls_dummyframe" onsubmit="subscribeClicked()">
-<input name="email" type="text"/><button onclick="subscribeClicked()" type="button">Subscribe/Confirm and Download</button>
+Consider subscribing to <a href="https://sourceforge.net/projects/fontforge/lists/fontforge-announce">the mailing list</a>.m<button onclick="downloadClicked()" type="button">Confirm and Download</button>
 </form>
 </div>
 <div id="dl_continue_box" style="display: none;">
-Thanks. Continue <a href="../gnulinux-dl">this way</a>.
+Continue <a href="../mac-dl">this way</a>.
```
Comment (jtanx) [inline en-US/downloads/gnulinux.md]: gnulinux->mac?

**Comment by jtanx** [comment]
> Collecting lots of data is bad, but we do need to get e-mail addresses somehow. 

I've always found that process pretty dubious, I'm actually against what we(?)'re doing. Sure, promote subscribing to the mailing list, but don't make it a(n apparently) necessary part of downloading the software.

This part too (which I think gets blocked anyway)
> http://mm.fontforge.org/ffml/presubscribe.php

Also just looks so dodgy. You're sending personal information over an *unencrypted* connection. What does it even do with that email address? Is it storing it? Who has access to that list? Why do we need to keep this? There's literally no oversight into what's going on with that.

As a side note from that, I find it *odd* that http://mm.fontforge.org is apparently hosting some other website.

**Comment by frank-trampe** [comment]
@jtanx, the mm server goes back to when SourceForge cut access to the lists. I knew that we would need to switch over and thus to have the e-mail addresses. I manage the server in question, and only a few other trusted people have any internal access to it. The lack of encryption is indeed problematic (or was when that feature worked), but I had originally hoped that it would last for only a few weeks.

As for why having the e-mail list is important, we may not be competing against forks anymore, but we are competing against older versions of FontForge, and I'm not sure that we've yet overtaken the 2012 release. Given the volume of improvements to existing functionality that we push through, it's almost tragic if somebody falls behind.

File: en-US/downloads/gnulinux.md (hunk 1 above)
Comment (frank-trampe) [reply en-US/downloads/gnulinux.md]: Oops. I'll fix that now.

**Comment by jtanx** [comment]
I still don't think that's the right way to go about it.

At the least, the submission code to the mm server should be removed.

**Comment by frank-trampe** [comment]
The code in question is not called anywhere right now.

**Pull request merged**